scipy/optimize/_linprog_ip.py

```diff
@@ -676,6 +676,9 @@ def _ip_hsd(A, b, c, c0, alpha0, beta, maxiter, disp, tol, sparse, lstsq,
                 The number of iterations performed.
             message : str
                 A string descriptor of the exit status of the optimization.
+    postsolve_args : tuple
+        Data needed by _postsolve to convert the solution to the standard-form
```

Comment:
should this be "**from** the standard-form problem"?

Reply:
The solution to the problem
vs
The solution from the problem
I don't think so.

Reply:
The way I read this original was:

Data needed by _postsolve to convert the solution [to -> from] the standard-form problem [back] into the [~~the~~] solution to the original problem.

As in _postsolve_args takes the standard-form problem and converts it to the original. But what you've said is correct. _postsolve takes the **solution**, not the standard form problem

Reply:
Ok, gotcha.